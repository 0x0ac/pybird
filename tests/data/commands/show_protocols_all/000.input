
0001 BIRD 1.3.0 ready.
2002-name     proto    table    state  since       info
1002-device1  Device   master   up     Jun13       
1006-  Preference:     240
  Input filter:   ACCEPT
  Output filter:  REJECT
  Routes:         0 imported, 0 exported, 0 preferred
  Route change stats:     received   rejected   filtered    ignored   accepted
    Import updates:              0          0          0          0          0
    Import withdraws:            0          0        ---          0          0
    Export updates:              0          0          0        ---          0
    Export withdraws:            0        ---        ---        ---          0

1002-P_PS1    Pipe     master   up     Jun13       => T_PS1
1006-  Preference:     70
  Input filter:   <NULL>
  Output filter:  <NULL>
  Routes:         0 imported, 0 exported
  Route change stats:     received   rejected   filtered    ignored   accepted
    Import updates:              0          0          0          0          0
    Import withdraws:            0          0        ---          0          0
    Export updates:              0          0          0          0          0
    Export withdraws:            0          0        ---          0          0

1002-PS1      BGP      T_PS1    start  Jun13       Passive       
1006-  Description:    Peering AS8954 - InTouch
  Preference:     100
  Input filter:   ACCEPT
  Output filter:  ACCEPT
  Routes:         0 imported, 0 exported, 0 preferred
  Route change stats:     received   rejected   filtered    ignored   accepted
    Import updates:              0          0          0          0          0
    Import withdraws:            0          0        ---          0          0
    Export updates:              0          0          0        ---          0
    Export withdraws:            0        ---        ---        ---          0
  BGP state:          Passive

1002-P_PS2    Pipe     master   up     14:20       => T_PS2
1006-  Preference:     70
  Input filter:   <NULL>
  Output filter:  <NULL>
  Routes:         0 imported, 0 exported
  Route change stats:     received   rejected   filtered    ignored   accepted
    Import updates:              0          0          0          0          0
    Import withdraws:            0          0        ---          0          0
    Export updates:              0          0          0          0          0
    Export withdraws:            0          0        ---          0          0

1002-PS2      BGP      T_PS2    start  14:20       Established       
1006-  Description:    Peering AS8954 - InTouch
  Preference:     100
  Input filter:   ACCEPT
  Output filter:  ACCEPT
  Routes:         24 imported, 23 exported, 0 preferred
  Route change stats:     received   rejected   filtered    ignored   accepted
  Import updates:             12          0          0          0         12
  Import withdraws:            3          0        ---          0          3
  Export updates:             12         12          0        ---          0
  Export withdraws:            3        ---        ---        ---          0
    BGP state:          Established
      Session:          external route-server AS4
      Neighbor AS:      8954
      Neighbor ID:      85.184.4.5
      Neighbor address: 2001:7f8:1::a500:8954:1
      Source address:   2001:7f8:1::a519:7754:1
      Neighbor caps:    refresh AS4
      Route limit:      9/1000
      Hold timer:       112/180
      Keepalive timer:  16/60
    
0000 
