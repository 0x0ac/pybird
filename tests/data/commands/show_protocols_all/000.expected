[{"export_updates_received": 0, "protocol": "BGP", "export_updates_rejected": 0, "import_updates_ignored": 0, "import_withdraws_rejected": 0, "export_updates_accepted": 0, "import_updates_accepted": 0, "import_updates_received": 0, "export_updates_filtered": 0, "import_withdraws_ignored": 0, "routes_imported": 0, "name": "PS1", "import_updates_filtered": 0, "export_withdraws_accepted": 0, "import_withdraws_accepted": 0, "export_withdraws_received": 0, "up": false, "state": "Passive", "routes_exported": 0, "import_updates_rejected": 0, "last_change": "2016-06-13T00:00:00", "import_withdraws_received": 0}, {"router_id": "85.184.4.5", "export_updates_received": 12, "protocol": "BGP", "export_updates_rejected": 12, "import_updates_ignored": 0, "import_withdraws_rejected": 0, "export_updates_accepted": 0, "import_updates_accepted": 12, "import_updates_received": 12, "export_updates_filtered": 0, "import_withdraws_ignored": 0, "routes_imported": 24, "name": "PS2", "import_updates_filtered": 0, "export_withdraws_accepted": 0, "import_withdraws_accepted": 3, "export_withdraws_received": 3, "up": true, "state": "Established", "routes_exported": 23, "import_updates_rejected": 0, "last_change": "2016-11-10T14:20:00", "import_withdraws_received": 3}]
